src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,83 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './Tag.module.css';
+import Text from '../Text';
+
+export type Color =
+  | 'neutral'
+  | 'alert'
+  | 'success'
+  | 'warning'
+  | 'revise'
+  | 'brand';
+
+export const stylesByColor = {
+  neutral: styles[`tag--color-neutral`],
+  warning: styles[`tag--color-warning`],
+  brand: styles[`tag--color-brand`],
+  alert: styles[`tag--color-alert`],
+  revise: styles[`tag--color-revise`],
+  success: styles[`tag--color-success`],
+};
```

Comment:
Brought over from v0. I assume the use case for this was to export the tag styles around but it's only being used in the story file as a list of colors, could we remove and replace in the story file as a simple enum/array?

Reply:
I personally like the deduplication mostly because that means if we change the props here at some point (like we add a new one), it's automatically updated in storybook as well.

Reply:
`stylesByColor` isn't being used to type props tho, and I think this problem is part of Andrew's todo haha

Reply:
I feel ambivalent haha. If we declared the color array in `Tag.tsx` I'm pretty sure you could then generate the types, i.e.:
```
export const COLORS = [...] as const;

export type Color = typeof COLORS[number];
```
per https://melvingeorge.me/blog/convert-array-into-string-literal-union-type-typescript

Reply:
Ah I like that approach, will use that, update story file, and remove Andrew todo comment